apps/src/lib/levelbuilder/rubrics/RubricsContainer.jsx

```diff
@@ -1,9 +1,99 @@
-import React from 'react';
+import React, {useState} from 'react';
+import PropTypes from 'prop-types';
+import {BodyTwoText, Heading1} from '@cdo/apps/componentLibrary/typography';
+import LearningGoalItem from './LearningGoalItem';
+import Button from '@cdo/apps/templates/Button';
+
+export default function RubricsContainer({unit, lesson}) {
+  // note that the use of currentId here is temporary until we are connected to the data
+  const [currentId, setCurrentId] = useState(1);
+  const [learningGoalList, setLearningGoalList] = useState([{id: currentId}]);
+
+  // holding levels in this temporary variable before connecting the data
+  const tempLevelList = [1, 2, 3, 4, 5];
+
+  const renderLearningGoalItems = learningGoalList.map(goal => (
+    <LearningGoalItem
```

Comment:
I'm surprised React didn't get grumpy about needing a `key` prop. When you render components using `map`, you need a `key` prop so that React can keep track of it ([docs link](https://react.dev/learn/rendering-lists#keeping-list-items-in-order-with-key)). In this case, it can just be `goal.id` :)

Reply:
Thanks for linking to that documentation!